src/app/navbar/navbar.component.html

```diff
@@ -11,16 +11,15 @@ <h1 class="h4 logo">
          *ngIf="authService.isAuthenticated()"
          dropdown
          [(isOpen)]="isSettingsDropdownOpen">
+      <a class="button button-link" href="//{{ apiHost }}">API</a>
```

Comment:
Should this link be a `target="_blank"`?

Reply:
Hm yea probably

Reply:
I generally lean against `target="_blank"`, [it has poor UX implications](https://ux.stackexchange.com/questions/5397/whats-your-take-on-target-blank) and generally if users want to open items in new tabs they know how to do that, and with `target="_blank"` there's no way for them to not open a new tab.

Reply:
An argument for and against. @jfrankl I'll leave you to adjust it later if desired but its probably fine as is for now.